README.md

```diff
@@ -23,6 +23,8 @@ This package implements some generic data structures.
 * [`rope`](./rope): a generic rope, which is similar to an array but supports efficient
   insertion and deletion from anywhere in the array. Ropes are typically used
   for arrays of bytes, but this rope is generic.
+* [`prope`](./prope): a persistent version of the rope, which allows for keeping different
+  versions of the rope without little extra time or memory.
```

Comment:
This should be "with only a little extra time or memory" right?

Reply:
Yes, thank you for the correction! I just fixed it, alongside the wrong gomarkdoc repo url.